ftplugin/vader.vim

```diff
@@ -49,3 +56,9 @@ augroup vader_syntax
   " autocmd FileType <buffer> call vader#syntax#include(1, '$')
 augroup END
 
+if exists('b:undo_ftplugin')
```

Comment:
No, it was based on tpope's haml ftplugin, where other ftplugins are sourced in-between.